opentasks/src/main/java/org/dmfs/tasks/ViewTaskFragment.java

```diff
@@ -594,13 +594,24 @@ else if (itemId == R.id.pin_task)
 			persistTask();
 			return true;
 		}
+		else if (itemId == R.id.share_task)
+		{
+			shareTaskDescription();
```

Comment:
No, we don't. (I inlined it)

I guess it's a matter of programming style, and I hope I won't drive you crazy with it :)
This case is not a really good example, but how I am looking at it usually is that for example here onOptionsItemSelected() method should only be concerned what happens on certain button clicks, not how that happens. That's a detail that is not relevant on his level. So if I want to put it fancily, methods have sort of abstraction levels as well, and also responsibilities.
If we don't apply single responsibility on this granularity as well, then I think methods tend to grow big and hard to follow, juggling with a lot of different kind of low level stuff.
(Again this concrete example is not the best, because the one-liner and also the conditions in onOptionItemsSelected() are straightforward, so sorry if this was irrelevant.)